backend/src/attempts/attempts.controller.ts

```diff
@@ -44,16 +43,10 @@ export class AttemptsController {
       where: { user_id },
```

Comment:
@escbooster12-dev 
- attempts service sir is the responsible for saving the attempt of the user  and  the user's taken category. 
- For the **isDone** , I created that one since it is a post request and needed a body that is why is used isDone boolean to identify the current user what category did he take